transcom/mymove
# B 20164 int - V2

## [20164](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A959436)
## [Original Integration Review](https://github.com/transcom/mymove/pull/12739)

## Summary

This is a followup to the original B-20164 review. The only change here is to update the UI to display the full address instead of the postal code.

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

1. Access the Customer app
2. Login as a customer
3. Fill in any required profile fields and create a new move
4. Fill in all the required fields for a move.
5. Add a new ppm shipment
6. Fill in all fields
7. When you hit the estimated incentive page
8. Verify that the full address is displayed for both starting and destination

### Frontend


### Backend


### Database

#### Any new migrations/schema changes:


## Screenshots
![Screenshot 2024-06-04 at 3 36 40 PM](https://github.com/transcom/mymove/assets/162157577/95e846c2-6bf6-4542-bf75-46f422848274)
